sql/moz-fx-data-shared-prod/firefox_accounts_derived/docker_fxa_admin_server_sanitized_v1/query.sql

```diff
@@ -18,4 +18,4 @@ SELECT
 FROM
   `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_admin_server_20*`
```

Comment:
These all need to be changed as well to e.g. `docker_fxa_admin_server` (or `docker_fxa_admin_server_v1` if there's going to be versioning, not sure)